scrapy/extensions/feedexport.py

```diff
@@ -350,7 +351,12 @@ def _start_new_batch(self, batch_id, uri, feed_options, spider, uri_template):
         :param uri_template: template of uri which contains %(batch_time)s or %(batch_id)d to create new uri
         """
         storage = self._get_storage(uri, feed_options)
-        file = storage.open(spider)
+
+        if "postprocessing" in feed_options:
+            file = PostProcessingManager(feed_options["postprocessing"], storage.open(spider), feed_options)
+        else:
+            file = storage.open(spider)
```

Comment:
- `file` definition could be simplified, and made slightly more readable.
- Seeing it in action now, I see it probably makes sense to simplify the signature of `PostProcessingManager`.

```suggestion
        file = storage.open(spider)
        if "postprocessing" in feed_options:
            file = PostProcessingManager(file, feed_options)
```